<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>281</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="headBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="chainBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="braceletBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="UpdateButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Info</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Общая стоимость:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Ignored</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Actorimg">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>567</width>
      <height>504</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>567</width>
      <height>504</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Desktop/timosh.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="chain">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>120</width>
      <height>171</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>190</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="bracelet">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>436</y>
      <width>170</width>
      <height>90</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>170</width>
      <height>90</height>
     </size>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="head">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>36</y>
      <width>280</width>
      <height>190</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>966</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
